plugins/catalog-backend/src/ingestion/processors/AwsOrganizationProcessor.test.ts

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { AwsOrganizationProcessor } from './AwsOrganizationProcessor';
+
+describe('AwsOrganizationProcessor', () => {
+  describe('readLocation', () => {
+    const processor = new AwsOrganizationProcessor();
+    const location = { type: 'aws-organization', target: 'b' };
+    const emit = jest.fn();
+    const listAccounts = jest.fn();
+
+    processor.organizations.listAccounts = listAccounts;
+    afterEach(() => jest.resetAllMocks());
+
+    it('generates component entities for accounts', async () => {
+      listAccounts.mockImplementation(() => {
+        return {
+          async promise() {
+            return {
+              Accounts: [
+                {
+                  Arn:
+                    'arn:aws:organizations::192594491037:account/o-1vl18kc5a3/957140518395',
+                  Name: 'testaccount',
+                },
+              ],
+              NextToken: undefined,
+            };
+          },
+        };
+      });
+      await processor.readLocation(location, false, emit);
+      expect(emit).toBeCalledWith({
+        type: 'entity',
+        location,
+        entity: {
+          apiVersion: 'backstage.io/v1alpha1',
+          kind: 'Component',
```

Comment:
A cloud accounts sounds a lot like a [Resource](https://github.com/backstage/backstage/issues/390), not like a Component which is more like a software entity. Resource is [not yet added](https://backstage.io/docs/features/software-catalog/descriptor-format#kind-resource) though, which is a bit unfortunate ... Hm.

Maybe we could continue using Component here just to unblock you, but with a clear plan to move over ASAP to using a Resource kind instead when one is available.

Reply:
Yeah, agreed - I think eventually it'd make sense for AWS accounts, GCP projects, and Azure Subscriptions to all end up as Resource objects with similar traits